Frameworks/UIKit/StarboardXaml/CompositorInterface.mm

```diff
@@ -332,7 +333,12 @@ void SetParams(UIFont* font,
         _isBold = (mask & UIFontDescriptorTraitBold) > 0;
         _isItalic = (mask & UIFontDescriptorTraitItalic) > 0;
         std::wstring wideBuffer = Strings::NarrowToWide<std::wstring>(text);
-        ConstructGlyphs([[font fontName] UTF8String], wideBuffer.c_str(), wideBuffer.length());
```

Comment:
The method returns the "correct" value, which is taken directly from the font file as I understand it. The issue is Xaml expects the family name to be a different value than the one on the file in some cases.  A developer may expect familyName to be the actual value from the file, in which case changing it in the CoreText/UIKit calls would be incorrect.